Header header
Point2D[] points
Point2D[] reprojectedPoints
geometry_msgs/Point[] points3D
geometry_msgs/Pose pose
